acceptance/cases/migration/change_table_test.rb

```diff
@@ -28,57 +28,6 @@ def with_change_table
         yield ActiveRecord::Base.connection.update_table_definition(:delete_me, @connection)
       end
 
-      def test_references_column_type_adds_id
```

Comment:
Most of these tests have been replaced with tests that use the mock server instead. The tests in this file were not real end-to-end tests anyways, as they only tested whether a specific method was called. That kind of testing is not very useful, as many of those method definitions are slightly different in the different supported versions.